src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,264 @@
+package duke;
+
+
+import duke.cmds.Cmds;
+import duke.deadline.Deadline;
+import duke.duke_exception.DukeException;
+import duke.event.Event;
+import duke.event.ToDo;
+import duke.task.Task;
+import netscape.javascript.JSObject;
+
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+import org.json.simple.JSONArray;
+import org.json.simple.JSONObject;
+import org.json.simple.JSONValue;
+
+public class Duke {
```

Comment:
There are little or no comments in across the files. Consider including [javadocs comments ](https://se-education.org/guides/conventions/java/basic.html#comments)to summarize about certain methods.